### PR TITLE
cmds/core/strace: add -o option

### PR DESCRIPTION
**Pull request by mpictor**
Signed-off-by: Mark Pictor <mpictor@gmail.com>

### CODE REVIEW

File: cmds/core/strace/strace.go [hunk 1]
```diff
@@ -25,15 +25,17 @@ import (
 	"github.com/u-root/u-root/pkg/strace"
 )
 
-var (
```
Comment (mpictor) [inline cmds/core/strace/strace.go:28]: Any reason for cmdUsage to be a var rather than const?

Comment (hugelgupf) [reply cmds/core/strace/strace.go:28]: nope not really

File: cmds/core/strace/strace.go [hunk 2]
```diff
@@ -25,15 +25,17 @@ import (
...
+	defaultOut = "<stdout>"
+	cmdUsage   = "Usage: strace [-o <outputfile>] <command> [args...]"
 )
 
 func usage() {
 	log.Fatalf(cmdUsage)
 }
 
 func main() {
+	o := flag.String("o", defaultOut, "write output to file")
```
Comment (hugelgupf) [inline cmds/core/strace/strace.go:38]: I'd just make this `"o", "", "Write output to file (if empty, stdout)"`

File: cmds/core/strace/strace.go [hunk 3]
```diff
@@ -44,7 +46,16 @@ func main() {
 	c := exec.Command(a[0], a[1:]...)
 	c.Stdin, c.Stdout, c.Stderr = os.Stdin, os.Stdout, os.Stderr
 
-	if err := strace.Strace(c, os.Stdout); err != nil {
+	out := os.Stdout
+	if *o != defaultOut {
```
Comment (hugelgupf) [inline cmds/core/strace/strace.go:50]: and if `len(*o) > 0`

File: cmds/core/strace/strace.go (hunk 2 above)
Comment (mpictor) [reply cmds/core/strace/strace.go]: done

**Pull request merged**